src/pe_reports/metrics.py

```diff
@@ -558,10 +554,27 @@ def dark_web_tags(self):
         dark_web_tags = dark_web_tags.rename(columns={"tags": "Tags"})
         return dark_web_tags
 
+    def alerts_site(self):
+        """Get alerts in invite-only markets."""
+        alerts = self.alerts
+        alerts_site = alerts[["site"]]
```

Comment:
I'd consolidate these lines to get rid of the `alerts` variable:
```suggestion
        alerts_site = self.alerts[["site"]]
```

Reply:
Addressed [here](https://github.com/cisagov/pe-reports/pull/114/commits/b3248fc1023126a5771b66f8f8e8d5c73048cece).